astropy/time/tests/test_basic.py

```diff
@@ -26,12 +26,15 @@
 except ImportError:
     HAS_PYTZ = False
 
-allclose_jd = functools.partial(np.allclose, rtol=2. ** -52, atol=0)
-allclose_jd2 = functools.partial(np.allclose, rtol=2. ** -52,
-                                 atol=2. ** -52)  # 20 ps atol
-allclose_sec = functools.partial(np.allclose, rtol=2. ** -52,
-                                 atol=2. ** -52 * 24 * 3600)  # 20 ps atol
-allclose_year = functools.partial(np.allclose, rtol=2. ** -52,
+
+FLOAT_EPS = np.finfo(float).eps
```

Comment:
Why the FLOAT_EPS rather than the other? It's not many more characters and it's less shouty.

Reply:
"the other"? you mean lower case, or `eps_float`?

Reply:
I mean `np.finfo(float).eps`, it's really not longer, it's immediately apparent what it is, and it shows that it's explicitly a function of the type `float`.

Reply:
OK, I see your point. I changed it everywhere.